app/views/pages/hiring_fundraiser.html.haml

```diff
@@ -0,0 +1,120 @@
+#page-hiring-job.bg-backgroundColorA.pb-16
+  %header
+    .lg-container.c-shapes
+      = link_to hiring_page_path, class: "c-prominent-link" do
+        = graphical_icon "arrow-left"
+        Back to All Job Listings
+
+      .role
+        = graphical_icon "hiring-fundraiser", category: 'graphics', css_class: 'graphic'
+        .information
+          %h1.text-h0.mb-12 Head of Partnerships and Fundraising
+          .details
+            .remote-tag
+              = graphical_icon "stars"
+              100% remote
+            .salary Salary: £50k (~$68k, ~€60k)
+
+  %article
+    .sm-container
+      %p.text-p-2xlarge.mb-40
+        %strong Become Exercism's Head of Partnerships and Fundraising!
+        Are you someone who can both think strategically, and do the hard work of finding and closing sponsorship and partnership deals?
+        If so, this role might be perfect for you. Come work with an exceptional team and help fix programming education.
+
+      %h2.text-h2.mb-12 Summary
+      %p.text-p-xlarge.mb-16
+        Exercism is a not-for-profit organisation providing world-class programming education to anyone for free, through a mixture of self-directed learning and mentoring.
+        We have grown to nearly a million members, purely by word-of-mouth, and are recognised as one of the leading education platforms online.
+        Exercism is built by a mixture of our core team and thousands of volunteers.
```

Comment:
If you want to stress diversity it might help to also mention that the current group is diverse already, e.g. "and a diverse group of thousands of volunteers."